Jenkinsfile

```
@@ -271,3 +319,7 @@ def publishChocoPackage(version, jfrogCliRepoDir, architectures) {
         }
     }
 }
+
+def dockerLogin(){
+  sh "docker login $REPO21_URL -u=ecosystem -p=$JFROG_CLI_AUTOMATION_ACCESS_TOKEN"
```

Using `--password-stdin` is [considered](https://docs.docker.com/engine/reference/commandline/login/#provide-a-password-using-stdin) more safe:
```suggestion
  sh "echo $JFROG_CLI_AUTOMATION_ACCESS_TOKEN | docker login $REPO21_URL -u=ecosystem --password-stdin"
```